### PR TITLE
The Conductor test is failing- any ideas?

### PR DESCRIPTION
**Pull request by anthonysexton**

### CODE REVIEW

File: train.rb [hunk 1]
```diff
@@ -1,15 +1,28 @@
...
+  def initialize(message_board)
+    @message_board = message_board
+  end
+
+  def acknowledge_slow_down!
+    message_board.confirm_slow_down!
+  end
+end
+
+class MessageBoard
```
Comment (jwo) [inline train.rb:27]: So, the MessageBoard class is empty --- do you think it will blow up when the `.confirm_slow_down!` is called?

**Comment by jwo** [comment]
Check for a spelling of `messsage_board.slow_down!` (3 letter s) => that'll fix your test

File: train.rb (hunk 1 above)
Comment (anthonysexton) [inline train.rb:27]: But weren't we able to leave the engineer class empty initially?   

Wouldn’t it also blow up when the conductor calls message_board.slow_down! ?

Comment (jwo) [inline train.rb:27]: > Wouldn’t it also blow up when the conductor calls message_board.slow_down! ?

What's the next step? how can message_board not blow up? and what tests are necessary?

Comment (anthonysexton) [inline train.rb:27]: I have no idea.  

Will doing Ruby Blocks help me here?

Comment (jwo) [inline train.rb:27]: I think the MessageBoard would have to know about the possible engineers, and the possible conductors. But maybe, the engineer could register with the message_board --

``` ruby
# somewhere inside the engineer
board.on_slow_down do {|message|
  confirm_slow_down!
  log_message(message)
end
```

(or something)

**Pull request closed without merging**